integration-tests/go.mod

```diff
@@ -14,6 +14,13 @@ require (
 	github.com/smartcontractkit/chainlink/v2 v2.1.0-beta0.0.20230427051455-edb338fd2536
 )
 
+require (
+	github.com/hashicorp/go-hclog v1.2.2 // indirect
+	github.com/hashicorp/go-plugin v1.4.9 // indirect
+	github.com/hashicorp/yamux v0.0.0-20200609203250-aecfd211c9ce // indirect
+	github.com/oklog/run v1.1.0 // indirect
+)
+
 require (
```

Comment:
```suggestion
```

Reply:
There's still an odd separate `// indirect` block here.

Reply:
i thought it would have been fixed magically my one of my many go mod tidy incantations. surprised that it wasn't. should be fixed now

Reply:
yeah the tooling doesn't undo these, and it occasionally introduces them for some reason :confused: